test/Main.hs

```diff
@@ -0,0 +1,14 @@
+module Main where
+
+import Test.Tasty (defaultMain, testGroup)
+import TestParser qualified
+import TestTemplate qualified
+
+main :: IO ()
+main =
+  defaultMain $
+    testGroup
+      "Lhx tests"
+      [ TestParser.tests
+      , TestTemplate.tests
+      ]
```

Comment:
Mandatory newline at the end of file is a good thing. Look at your editor's settings: it should have such option.